File: python3.9/1010.py
LIST1 = input().split(" ")
LIST2 = input().split(" ")
COD_P1, NUMERO_P1, VALOR_P1 = LIST1
COD_P2, NUMERO_P2, VALOR_P2 = LIST2

CALC = (int(NUMERO_P1) * float(VALOR_P1)) + (int(NUMERO_P2) * float(VALOR_P2))

print("VALOR A PAGAR: R$ {:.2f}".format(CALC))

File: Codigos-C/1010.c
#include <stdio.h>
 
int main() {
 
	int cod_p1, num_p1, cod_p2, num_p2;
	float valor_p1, valor_p2, total;
	
	scanf("%d %d %f",&cod_p1,&num_p1,&valor_p1);
	scanf("%d %d %f",&cod_p2,&num_p2,&valor_p2);
		
	total = (num_p1 * valor_p1) + (num_p2 * valor_p2);
	
	printf("VALOR A PAGAR: R$ %.2f\n", total);
 
    return 0;
}
